seed/php-sdk/imdb/src/Types/WeatherReport.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Seed\Types;
+
+enum WeatherReport: string {
```

Comment:
I'm back to having this an actual `enum` (rather than string consts) 🙃 

The IDE complains either way if I mark the type as this in the PHPDoc. I think users can figure out to do `.value` if they use the enum, and enum allows easier `.match` authorimg

Reply:
Should we do the same thing for environments, or should we special case that? I was originally thinking we'd have something like:

```php
class Environments {
  public const PRODUCTION = "https://acme.co";
  public const STAGING = "https://acme-staging.co";
}
```

Happy to model this as an enum if we want though, especially because most people won't even interact with it - the default environment will normally be set for them:

```php
enum Environments: string {
  case PRODUCTION = "https://acme.co";
  case STAGING = "https://acme-staging.co";
}
```

Reply:
No strong feelings! enum seems reasonable to me with the first class support for string-backed, but as you said, not much user interaction!